### ISSUE TITLE
Request: Don't display bullets in front of checkboxes

### ISSUE THREAD

**Issue by necopinus** [dvorka/mindforger#886]
Quick request: When using lists with check boxes (`- [ ] foo`, `- [X] bar`), don't render the leading bullet point.

This is honestly a personal preference of mine, but does better match how GitHub handles check boxes. For example, 

```markdown
- [ ] foo
- [ ] bar
- [X] baz
```

renders as:

- [ ] foo
- [ ] bar
- [X] baz

**Comment by dvorka** [comment]
Makes sense :+1: MindForger uses `discount` as MD to HTML rendering library - I will need to fix it there.
